common/src/main/java/org/openmrs/analytics/FhirStoreUtil.java

```diff
@@ -91,7 +92,14 @@ public MethodOutcome uploadResource(Resource resource) {
 			interceptors = Collections.singleton(new BasicAuthInterceptor(sinkUsername, sinkPassword));
 		}
 		
-		return uploadBundle(sinkUrl, bundle, interceptors);
+		Collection<MethodOutcome> responses;
+		try {
```

Comment:
@bashir2   , i ddi this here .
the pipeline was failing with an error "ResourceVersionConflictException  Multiple clinets trying to create same resources with self assigned ID"  , but on reruning it again it worked,

so what i did here is to catch the exception and re-upload the bundle

Reply:
cc @kimaina